### ISSUE TITLE
NTR: Pancreatic biopsy: necrosis or atrophy (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9209]
New term request Pancreatic biopsy: necrosis or atrophy (elthat_101205180222):   UMLS: 0267953

**Comment by pnrobinson** [comment]
@MickeySegal
-- this is "bundled". I added the following. Please open a new issue if necrosis is required.

Adding new term: 
- ID: HP:6000156
- label: Pancreatic atrophy
- parents: [Abnormal pancreas morphology (HP:0012090)](https://hpo.jax.org/app/browse/term/HP:0012090)
- pmid: PMID:22787562
- def: Wasting (shrinking) of the pancreas.
- comment: Pancreatic atrophy is commonly seen in elderly patients and may also be a sequela of conditions including chronic pancreatitis, cystic fibrosis, and Shwachman-Bodian-Diamond syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9209
- synonyms:
